mobile/native/src/ln_dlc/mod.rs

```diff
@@ -48,6 +48,11 @@ pub async fn refresh_wallet_info() -> Result<()> {
     Ok(())
 }
 
+pub fn get_seed_phrase() -> Result<Vec<String>> {
+    let node = NODE.try_get().context("failed to get ln dlc node")?;
```

Comment:
I would argue that if we never manipulate / change it, we should just ignore `PoisonError` and be always reaching into the actual Seed. That will slightly simplify this code as there's no `Result<>`.

Reply:
Agreed! But then we should also adapt this on all other functions where we are accessing the node like that. (and I didn't want to do this now ;)

Reply:
Not sure if I get you, @klochowicz, but I would propose to always `expect` when accessing the `NODE` since it must be set at all times after startup (and if it isn't crashing the app is best).